### DISCUSSION TITLE
Is it possible to configure ECMA-VERSION-CHECK to check against ES8 ?

### DISCUSSION THREAD

**Discussion by Verthon** [web-infra-dev/rsdoctor#506] [Q&A] [answered]
Hello Team

rs-doctor version: `"@rsdoctor/webpack-plugin": "0.4.1"`

Our project only supports the latest browsers, while RSDoctor checks against ES5.
Could you let me know if there’s a way to configure this, or if this feature might be considered in a future release?

<img width="2449" alt="Screenshot 2024-09-09 at 09 56 38" src="https://github.com/user-attachments/assets/c160033f-dfc3-4721-9f14-efd27d82961d">

Thank you.

**Comment by yifancong** [comment, marked as answer]
OK, I'll add this features.  #532
